tck/features/MatchAcceptance.feature

```diff
@@ -177,8 +291,14 @@ Feature: MatchAcceptanceTest
 
   Scenario: Return two subgraphs with bound undirected relationship and optional relationship
     Given an empty graph
-      And having executed: CREATE (a:A {value: 1})-[:REL {name: 'r1'}]->(b:B {value: 2})-[:REL {name: 'r2'}]->(c:C {value: 3})
-    When executing query: MATCH (a)-[r {name:'r1'}]-(b) OPTIONAL MATCH (b)-[r2]-(c) WHERE r<>r2 RETURN a,b,c
+    And having executed: CREATE (a:A {value: 1})-[:REL {name: 'r1'}]->(b:B {value: 2})-[:REL {name: 'r2'}]->(c:C {value: 3})
+    When executing query:
+      """
+      MATCH (a)-[r {name:'r1'}]-(b)
+      OPTIONAL MATCH (b)-[r2]-(c)
+      WHERE r<>r2
```

Comment:
Added a PR to the style guide to always use padding space around operators. Do you agree?